libr/asm/p/asm_avr.c

```diff
@@ -412,11 +412,18 @@ RAsmPlugin r_asm_plugin_avr = {
 	.name = "avr",
 	.arch = "avr",
 	.license = "GPL",
-	.bits = 8|16,
+	.bits = 8|16|32|64,
```

Comment:
there's no 64bit avr, so drop this,. same for 32 and .. maybe 16 too, because the only 16 bit register is the pc and sp iirc